plugins/woocommerce-admin/client/core-profiler/index.tsx

```diff
@@ -100,6 +110,65 @@ const Extensions = ( {
 	);
 };
 
+const getAllowTrackingOption = async () =>
+	resolveSelect( OPTIONS_STORE_NAME ).getOption(
+		'woocommerce_allow_tracking'
+	);
+
+const handleTrackingOption = assign( {
+	optInDataSharing: (
+		_context,
+		event: DoneInvokeEvent< 'no' | 'yes' | undefined >
+	) => event.data !== 'no',
+} );
+
+const recordTracksIntroCompleted = () => {
+	recordEvent( 'storeprofiler_step_complete', {
+		step: 'store_details',
+		wc_version: getSetting( 'wcVersion' ),
+	} );
+};
+
+const recordTracksIntroSkipped = () => {
+	recordEvent( 'storeprofiler_store_details_skip' );
+};
+
+const recordTracksIntroViewed = () => {
+	recordEvent( 'storeprofiler_step_view', {
+		step: 'store_details',
+		wc_version: getSetting( 'wcVersion' ),
+	} );
+};
+
+const updateTrackingOption = (
+	_context: CoreProfilerStateMachineContext,
+	event: IntroOptInEvent
+) => {
+	if (
+		event.payload.optInDataSharing &&
+		typeof window.wcTracks.enable === 'function'
+	) {
+		window.wcTracks.enable( () => {
+			initializeExPlat();
+		} );
+	} else if ( ! event.payload.optInDataSharing ) {
+		window.wcTracks.isEnabled = false;
+	}
```

Comment:
I'm unfamiliar with this bit, when do we need to initialize explat?

Reply:
This function is copied from the existing OBW code. `initializeExPlat` was added in https://github.com/woocommerce/woocommerce-admin/pull/6669/. I think we initialize here in case we need to do any explat experiments.

https://github.com/woocommerce/woocommerce/blob/64dbeba26072f66e7f8f52e65aabc02161c21ba9/plugins/woocommerce-admin/client/profile-wizard/steps/usage-modal.js#L59-L83

Usually, we initialize explat when explat the package is loaded.

https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/explat/src/index.ts#L25-L31